packages/eslint-config-airbnb/rules/react.js

```diff
@@ -8,12 +8,22 @@ module.exports = {
   'rules': {
     // Prevent missing displayName in a React component definition
     'react/display-name': 0,
+    // Forbid certain propTypes (any, array, object)
+    'react/forbid-prop-types': 0,
```

Comment:
yes it does take a config...
I'll add the default config to all that have a config